lib/reducer.ts

```diff
@@ -70,23 +74,36 @@ export const propertiesIntrospectionFieldReducer:
 export const definitionsIntrospectionFieldReducer:
     MemoListIterator<IntrospectionFieldReducerItem, JSONSchema6Acc, ReadonlyArray<IntrospectionFieldReducerItem>> =
     (acc, curr: IntrospectionFieldReducerItem): JSONSchema6Acc => {
-
         if (isIntrospectionField(curr)) {
-
             const returnType = isNonNullIntrospectionType(curr.type) ?
                 graphqlToJSONType(curr.type.ofType) :
                 graphqlToJSONType(curr.type);
+
             acc[curr.name] = returnType;
         } else if (isIntrospectionInputValue(curr)) {
             const returnType = isNonNullIntrospectionType(curr.type) ?
                 graphqlToJSONType(curr.type.ofType) :
                 graphqlToJSONType(curr.type);
+
             acc[curr.name] = returnType;
+            if (curr.defaultValue) {
+                acc[curr.name].default = resolveDefaultValue(curr)
+            }
         }
+
         acc[curr.name].description = curr.description || undefined;
         return acc;
     };
 
+// ENUM type defaults will not JSON.parse correctly, so if it is an ENUM then don't
+// try to do that.
+// TODO: fix typing here
```

Comment:
I'm not a typing wizard at all 🧙  so I don't know how to fix this error/warning.